strategies/changed/src/main/java/org/arquillian/smart/testing/vcs/git/GitChangesDetector.java

```diff
@@ -75,7 +93,7 @@ private File findRepoRoot(File currentDir) {
             .map(diffEntry -> {
                 try {
                     final File sourceFile = new File(repoRoot, diffEntry.getNewPath());
-                    return new ClassNameExtractor().extractFullyQualifiedName(sourceFile);
+                    return ClassNameExtractor.extractFullyQualifiedName(sourceFile);
```

Comment:
Like @lordofthejars pointed above, `extractFullyQualifiedName` does not change across instances and as the `ClassNameExtractor` class follows `Single Responsibilty Principle`, it seems legit to make this `static` in my opinion.